docker/Dockerfile

```diff
@@ -66,10 +71,11 @@ RUN \
 	# Remove Git and clean apt cache
 	apt-get clean && \
 	apt-get remove -y git && \
-	apt-get -y autoremove
+	apt-get -y autoremove && \ 
+	chmod +x docker/entrypoint.sh
 
 USER node
 ENV NODE_ENV=production
 
-ENTRYPOINT ["pm2-runtime", "bin/iotagent-json"]
+ENTRYPOINT ["docker/entrypoint.sh"]
```

Comment:
Looking to Cygnus Dockerfile (https://github.com/telefonicaid/fiware-cygnus/blob/master/docker/cygnus-ngsi/Dockerfile) we have: 

```
ENTRYPOINT ["/cygnus-entrypoint.sh"]
```

So why we need `docker/` here? It will be better to be homogeneous and have:

```
ENTRYPOINT ["/iotagent-json-entrypoint.sh"]
```

Reply:
@fgalan As explained in comment https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r254773976 docker/entrypoint.sh was used.

Reply:
Cygnus adds an unnecessary `COPY` statement to ensure the `cygnus-entrypoint.sh` file can be found in the root of the container. There are two copies of the same file within the image.

```Dockerfile
# Copy some files needed for starting cygnus-common
COPY cygnus-entrypoint.sh /
COPY agent.conf ${FLUME_HOME}/conf/

# Define the entry point
ENTRYPOINT ["/cygnus-entrypoint.sh"]
```

OK in the case of the Cygnus repository, this may make sense given there are multiple `Dockerfiles`, but for a simple repo with one `Dockerfile` such as this the duplication is unnecssary

Reply:
As entrypoint.sh script is written inside a docker folder so I feel ENTRYPOINT ["docker/entrypoint.sh"] would be a better option. What do you think @fgalan @jason-fox ?

Reply:
Ok. Makes sense. Let's keep entrypoint.sh file within docker/

NTC